store/src/java/com/zimbra/cs/account/soap/SoapDistributionList.java

```diff
@@ -81,11 +81,31 @@
                 SoapProvisioning.getAttrs(e), prov);
         addDlm(e, getRawAttrs());
     }
-
+    
+    
+    /**
+     * This method is modified as it is overriding the existing member/s value with zero against zimbraMailForwardingAddress attribute.
+     * So now checking if members list is zero than it should not override. (ZBUG-651)
+     * @param members This is list of members
+     * @param attrs This is list of attribute
+     * @return nothing
+     */
     private void addDlm(List <String> members, Map<String, Object> attrs) {
-        attrs.put(Provisioning.A_zimbraMailForwardingAddress,
+    	if (members.size() > 0)
+    		attrs.put(Provisioning.A_zimbraMailForwardingAddress,
                 members.toArray(new String[members.size()]));
     }
+    
+    /**
+     * This method is added against the change in addDlm(), which provides blank list of members.
+     * @param attrs to put/override attribute with blank member list.
+     * @return nothing
+     * ZBUG-651
+     */
```

Comment:
Please draft the comment starting with the bug number:
`ZBUG-651: This method is added against the change in addDlm(), which...`

Reply:
Done.